### ISSUE TITLE
[Bug]: Crash in feed listing of !Linux@lemmy.ml

### ISSUE THREAD

**Issue by dpoon** [Memmy-App/memmy#1073]
### Check for open issues

- [X] I have verified that another issue for this is not open, or it has been closed and has not been fixed.

### Minimal reproducible example

When viewing the feed listing for !Linux@lemmy.ml, version 1.1.2 (86) crashes.

To help identify the post that triggers the crash, I've changed to large fonts and non-compact view to reduce the number of posts shown. After varying the sort order and comparing against the feed in Lemmy's web interface, I'm suspecting "[Who uses pure GNOME (no extensions)](https://lemmy.ca/post/8829094)", because:

- It's currently at the top of the community when sorting by Hot, and when viewing Hot for !Linux@lemmy.ml it crashes right away
- There are currently 47 comments, and when sorting by Most Comments, you can't scroll beyond posts with ~47 comments — it just starts displaying emptiness.

### Expected Behavior

App shouldn't crash.

### Version

1.1.2 (86)

### App Store or TestFlight?

TestFlight

### iOS Version

16.x

### Device

iPhone 13 mini

**Comment by gkasdorf** [comment]
Ok, I repro'd it in the simulator. Odd that it's not happening on my device though. Let me look into it 👍

**Comment by gkasdorf** [comment]
😅 So this is fun! Went ahead and tried catching native errors and found the crash to be here:

<img width="966" alt="Screenshot 2023-11-07 at 11 07 16 AM" src="https://github.com/Memmy-App/memmy/assets/127079571/a93649db-b3d6-46ab-ac13-dcb430d48eab">

Well that's a great way to explain the error!

I do notice though that you're on iOS 16 and that's why I set up an iOS 16 simulator. Wondering if this was fixed in iOS 17 and that is why I don't get the crash there.

The image in question is this: https://upload.wikimedia.org/wikipedia/commons/1/1b/Linux_Distribution_Timeline.svg which does load fine for me on my device.

I suppose what I'll have to do for now is not render SVGs for users on iOS 16 or lower and instead just offer the link to the image in the browser (honestly shouldn't be a very common thing at all anyway). Might be able to use a different library to render the SVGs as well, so I'll look into that.

Thank you for the excellent repro!
